### PR TITLE
More strings

### PR DESCRIPTION
**Pull request by Noxturnix**
This PR introduces additional strings for 5 setting pages

- [x] Account setting strings
- [x] Experiment setting strings
- [x] Theme shop strings
- [x] Appearance setting strings
- [x] Native setting strings

Linked PR: https://github.com/revoltchat/revite/pull/381

### CODE REVIEW

File: en.json [hunk 1]
```diff
@@ -319,7 +364,9 @@
                     "reserved": "Note: This field is reserved for the future."
                 },
                 "theme_shop": {
-                    "title": "Theme Shop"
+                    "title": "Theme Shop",
+                    "under_construction_warning": "This section is under construction.",
```
Comment (insertish) [inline en.json:368]: This string shouldn't be translated.

**Review by insertish — CHANGES_REQUESTED** [review]

File: en.json (hunk 1 above)
Comment (Noxturnix) [reply en.json:368]: > This string shouldn't be translated.

Should [these 2 strings](https://github.com/revoltchat/translations/pull/21/files/3426cc9528d31bd3411b09af3167ec9771e44440#diff-6f6ea5e5e2c2676af6d6cbd64866a01813f39bf2632e0f893970fd774a7d4334R197-R198) (line 197-198) also be removed? I think they are both temporary strings like this one

Comment (Rexogamer) [reply en.json]: As far as I can tell, the 2FA strings should be okay to keep as 2FA is longer-term - in comparison, the theme shop may be out of beta a lot quicker

File: en.json [hunk 2]
```diff
@@ -293,12 +300,50 @@
...
-                    "not_available": "No experiments are available at this time."
+                    "not_available": "No experiments are available at this time.",
+                    "theme_shop": {
+                        "title": "Theme Shop",
+                        "description": "Allows you to access and set user submitted themes."
+                    },
+                    "search": {
+                        "title": "Search",
+                        "description": "Allows you to search for messages in channels."
+                    }
```
Comment (Rexogamer) [inline en.json:346]: I think this experiment has been removed and released unflagged? I've never personally seen a search experiment

**Comment by insertish** [comment]
Closing as-per https://github.com/revoltchat/revite/pull/381#issuecomment-1402362787

**Pull request closed without merging**